packages/stockflux-launcher/src/index.css

```diff
@@ -1,7 +1,14 @@
 :root {
+  --icon-active: #29E9E3;
+  --icon-inactive: #8A96A6;
+  --icon-hover-light: rgb(27,59,86, 0.15);
+  --launcher-color: #0D2437;
   --search-input-min-width: 250px;
-  --search-input-width: 400px;
+  --search-input-text-color: #FFFFFF;
+  --search-input-width: 396px;
+  --search-input-width-undocked: 360px;
   --search-input-height: 30px;
+  --titlebar-color: #071521;
```

Comment:
Some of these colors are already defined `packages\stockflux-components\src\index.css` and are repeated here. We should avoid this and strive for color declarations being defined only once so that if color changes, we would only need to change it in one place.